frontend/src/features/children/components/childDetails.jsx

```diff
@@ -0,0 +1,114 @@
+import {useNavigate, useParams} from "react-router-dom";
+import {ChildQuery} from "../queries";
+import React, {useEffect, useState} from "react";
+import {APP_ROUTES} from "../../../core/routes";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import {EditableTitle} from "../../../shared";
+import {FieldsQuery} from "../../fields/queries";
+
+export const ChildDetails = () => {
```

Comment:
I think this component should be called something like `EditChildDetails` / `UpdateChildDetails`. It should use `<form>` (look at  `childForm` comments )